lntest/itest/lnd_payment_test.go

```diff
@@ -374,31 +376,40 @@ func testAsyncPayments(net *lntest.NetworkHarness, t *harnessTest) {
 		}
 
 		return true
-	}, time.Second*5)
+	}, defaultTimeout)
 	if err != nil {
 		t.Fatalf("failed to assert alice's pending htlcs and/or remote/local balance")
 	}
 
 	// Wait for Bob to receive revocation from Alice.
-	time.Sleep(2 * time.Second)
+	err = wait.NoError(func() error {
+		ctxt, _ = context.WithTimeout(ctxb, defaultTimeout)
+		bobChan, err := getChanInfo(ctxt, net.Bob)
+		if err != nil {
+			t.Fatalf("unable to get bob's channel info: %v", err)
```

Comment:
nit: return this err (and in code below) as well.

Reply:
I think this is a deal breaker here, so we do want to fail instantly if we can't get the chan info vs we're attempting to get the expected numbers in a timeout.

Reply:
Ah, I see. Makes sense.